tests/FondOfSpryker/Shared/GoogleTagManagerStoreConnectorConstants.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace FondOfSpryker\Shared\GoogleTagManagerStoreConnector;
+
+interface GoogleTagManagerStoreConnectorConstants
+{
+    public const FIELD_CURRENCY = 'currency';
```

Comment:
is this needed in tests?

Reply:
its already marked as "outdated"

Reply:
yes, cause you pushed after i started my review

Reply:
yepp, sry, my fault